pygfx/renderers/wgpu/engine/renderer.py

```diff
@@ -321,6 +321,19 @@ def blend_mode(self):
         """
         return self._blend_mode
 
+    @staticmethod
+    def register_blend_mode(name, blender=None):
+        """Register a new blender for usage with rendering pipelines."""
```

Comment:
Let's document that this is experimental. Until now the blender was an implementation detail, this makes it a bit public. So let's make it use-at-your-own risk.

Reply:
> So let's make it use-at-your-own risk.

Do you want to use the convention of `_register_blend_mode` or just add enough comments?

Reply:
ok i tried to add maximum caution